core/ir/encode_api.h

```diff
@@ -50,6 +50,7 @@ typedef enum _dr_isa_mode_t {
     DR_ISA_ARM_A32,           /**< ARM A32 (AArch32 ARM). */
     DR_ISA_ARM_THUMB,         /**< Thumb (ARM T32). */
     DR_ISA_ARM_A64,           /**< ARM A64 (AArch64). */
+    DR_ISA_RV64IMAFDC,        /**< RISC-V (rv64imafdc). */
```

Comment:
Does this mean DR would not support an arch missing one of the letters?  DR itself probably doesn't care about the floating-point support, so could it support r64ima?  Or is it implied that r64imafdc includes r64ima?

Reply:
rv64imafdc includes rv64ima.

However given the flexible and extensible nature of RISC-V spec it could be theoretically possible to create rv64i ISA with ANY combination of other extensions.

Honestly though I'm not sure yet how to handle possible subsets of ISA extensions in this enum.

That said, the spec defines IMAFD as the standard extension for a general purpose CPU. I'm planning for "C" (compressed) because adding it later on may be more work.